test/e2e/flowaggregator_test.go

```diff
@@ -336,6 +339,12 @@ func checkBandwidthFromRecord(t *testing.T, record, bandwidth string) {
 	}
 }
 
+func checkFlowType(t *testing.T, record string, flowType uint8) {
+	if !strings.Contains(record, fmt.Sprintf("%s: %d", "flowType", flowType)) {
+		t.Errorf("Record does not have correct flowType")
+	}
```

Comment:
nit: there is an `assert.Contains` assertion

Reply:
Thanks, addressed.